### ISSUE TITLE
Authorization with SSO (Okta)

### ISSUE THREAD

**Issue by agzam** [ahungry/org-jira#96]
Can someone help me. Our JIRA instance access controlled with Okta. I've been trying to use this trick  https://github.com/ahungry/org-jira#authorization-workaround-not-secure but it's not working.

I set `jiralib-token` to be something like:

    '("cookie" . "long-long thing I copied from the browser")

and it still says: 

    REQUEST [error] Error (error) while connecting to https://jira.my-company.com/rest/api/2/search.
    
although if I use curl including just that header - it works:

    curl https://jira.my-company.com/rest/api/2/search -H "cookie: long-long thing I copied from the browser"

**Comment by ahungry** [comment]
I use:

```
       (setq jiralib-token
                 `("Cookie" . ,(format "<omitted browser cookie string>")))
```

**Comment by ahungry** [comment]
Any luck on this?  I wonder if the copy/paste itself may have had a problem?

**Comment by agzam** [comment]
Sorry for the late response, I didn't have much chance to dig into this. it seems at my org specifically, because things are behind Incapsula (security tool), that thing does some munging to the cookie and it the pieces required for the package to work need to be carefully extracted by hand. Even when I did so, I have failed to read projects and issues. There's something sketchy happening, but it might be due to how things just are set up here, nothing to do with the package.
I'm closing this issue for now. Thank you for your help!
